scripts/install-helpers/baremetal-coco/install.sh

```diff
@@ -369,6 +371,11 @@ function set_kernel_params_for_kata_agent() {
     kata_override="[hypervisor.qemu]
 kernel_params= \"agent.aa_kbc_params=cc_kbc::$trustee_url\""
 
+    if [ -n "$cluster_https_proxy" ]; then
+        kata_override="[hypervisor.qemu]
+kernel_params=\"agent.aa_kbc_params=cc_kbc::$trustee_url agent.https_proxy=$cluster_https_proxy agent.no_proxy=$cluster_no_proxy\""
+    fi
+
```

Comment:
What about concatenating `"agent.https_proxy=..."` to the existing `kernel_params` instead of duplicating the code ? And do the same for `agent.no_proxy`.

Reply:
Done, please, take a look.